app/models/equipment_model.rb

```diff
@@ -107,8 +107,14 @@ def not_associated_with_self
   validates_attachment_size :photo,
                             less_than: 1.megabytes,
                             message: 'must be less than 1 MB in size'
-  validates_attachment :documentation,
-                       content_type: { content_type: 'application/pdf' }
+  validates_attachment_content_type :documentation,
+                       content_type: ['application/pdf'],
+                       message: 'must be pdf'
+  
+  validates_attachment_size :documentation,
+                            less_than: 5.megabytes,
```

Comment:
I think 5 MB is reasonable, although storage is so cheap these days 10 MB is probably ok too. Say you have 200 equipment models, each with 10 MB documentation, that's still only 2 GB of data (which costs virtually nothing to host). On second thought, let's kick it up so no one complains to us about the limit :stuck_out_tongue: